docs/developer-docs/multi-chain/bitcoin/using-btc/btc-comparison.mdx

```diff
@@ -0,0 +1,82 @@
+---
+keywords: [intermediate, bitcoin, btc, icp]
+---
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Understanding ICP for Bitcoin developers
+
+<MarkdownChipRow labels={["Intermediate", "Bitcoin"]} />
+
+## Overview
+This guide is designed to help Bitcoin developers understand the core concepts of ICP for using ICP as a Layer 2 solution for Bitcoin and as a separate blockchain.
+
+### Smart contracts
+
+Bitcoin’s scripting language, Script, is fairly simple, which makes complex smart contracts rather difficult to deploy directly on the base layer. 
+
+Bitcoin developers who are looking to build more complex smart contracts will need to build on a Bitcoin smart contract layer such as ICP, each offering its unique set of developer tools. 
```

Comment:
```suggestion
Bitcoin developers who are looking to build complex smart contracts will need to use Bitcoin smart contract layer (referred to as a Layer-2 or L2), such as ICP. Each L2 solution offers a unique set of developer tools. 
```

Reply:
How about "Bitcoin developers who are looking to build complex smart contracts will need to use a Bitcoin smart contract layer"?